source/includes/_user_group.md

```diff
@@ -0,0 +1,284 @@
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "ldap_dn": null,
+    "name": "new_group",
+    "tenants": [],
+    "users": []
+}
+```
+
+`GET "{manager-ip}/api/v3/user-groups/{group-name}"`
+
+Gets a specific user.
+
+### URI Parameters
+* `group-name`: The name of the group to retrieve.
+
+### Response
+A `Group` resource.
+
+
+
+
+
+## List User Groups
+
+> Request Example
+
+```shell
+$ curl -X GET --header "tenant: default_tenant" -u user:password "http://<manager-ip>/api/v3/user-groups"
+```
+
+```python
+# Python Client-
+client.user_groups.list()
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': 'default_tenant'
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/user-groups",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "items":
+        [
+            {
+                "ldap_dn": null,
+                "tenants": [],
+                "name": "new_group",
+                "users": []
+            }
+        ],
+}
+```
+
+`GET "{manager-ip}/api/v3/user-groups"`
+
+List all user groups.
+
+### Response
+Field | Type | Description
+--------- | ------- | -------
+`items` | list | A list of `Group` resources.
+
+
+
+## Create User Groups
+
+> Request Example
+
+```shell
+$ curl -X POST -H "Content-Type: application/json" -H "tenant: default_tenant" -d '{"group_name": <group-name>, "ldap_group_dn": <optional-ldap-dn>}' -u user:password "http://<manager-ip>/api/v3/user-groups"
+```
+
+```python
+# Python Client-
+client.user_groups.create(group_name=<group-name>, ldap_group_dn=<optional-ldap-dn>)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': <tenant-name>
+}
+
+vat data = {
+    "group_name": <group-name>,
+    "ldap_group_dn": <optional-ldap-dn>
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/user-groups",
+  "method": "POST",
+  "headers": headers,
+  "data": data
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "ldap_dn': "group_ldap_dn",
+    "name": "new_group",
+    "tenants": [],
+    "users": []
+}
+```
+
+`POST -d '{"group_name": <group-name>, "ldap_group_dn": <optional-ldap-dn>}' "{manager-ip}/api/v3/user-groups"`
+
+Creates a new user group.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`group_name` | string | The group name.
+`ldap_group_dn` | string | Optional parameter, The distinguish name of corresponding LDAP group (if using LDAP).
```

Comment:
Optional parameter, The **distinguishing** name of **the** corresponding LDAP group **, if appropriate** ~~(if using LDAP).~~